### ISSUE TITLE
dropbox detection

### ISSUE THREAD

**Issue by faroit** [sigsep/share#9]

**Comment by ThreeDeeJay** [comment]
Currently it detects dropbox links, but it [doesn't seem to parse them properly](https://github.com/audiolabs/trackswitch.js/issues/11#issuecomment-1881764054).
For example, when sharing dropbox gives me a link like this:
https://www.dropbox.com/scl/fi/qcc1huj581v62rb64wko1/Stereo.flac?rlkey=6rjllecxo3o9414glrq8kzjgd&dl=0
and when I paste it into the Unmix URL box, it turns it into this:
https://dl.dropboxusercontent.com/s/fi/qcc1huj581v62rb64wko1
but that won't work (gets stuck loading when clicking Preview Track), so I have to manually copypaste the `rlkey` parameter manually, so this is what works:
https://dl.dropboxusercontent.com/s/fi/qcc1huj581v62rb64wko1?rlkey=6rjllecxo3o9414glrq8kzjgd

**Comment by faroit** [comment]
@ThreeDeeJay Yeah. They have changed the api at some point. Do you want to make a PR to change the code?

**Comment by ThreeDeeJay** [comment]
I had looked at the code but I'm not familiar with this language/format so I'm not sure how to fix the behavior 😔 
https://github.com/sigsep/share/blob/555fc9bd0fecd4a136bc17836c5074ee9aee0ee0/src/views/Create.vue#L78
